plutus-ledger/src/Ledger/Tx/Internal.hs

```diff
@@ -105,6 +153,22 @@ scriptTxIns = (\x -> folding x) . Set.filter $ \case
     TxIn{ txInType = Just ConsumeScriptAddress{} } -> True
     _                                              -> False
 
+newtype TxOut = TxOut (C.TxOut C.CtxTx C.BabbageEra)
+  deriving stock (Show, Eq, Generic)
+  deriving anyclass (ToJSON, FromJSON)
+
+instance Serialise TxOut where
+  encode = undefined -- FIXME
+  decode = undefined -- FIXME
+
+instance NFData TxOut where
+  rnf = undefined -- FIXME
+
+instance OpenApi.ToSchema TxOut where
+  declareNamedSchema = undefined -- FIXME
+
+instance Pretty TxOut where
+  pretty = viaShow
```

Comment:
Notice this is still WIP

Reply:
All these instances have been temporarly removed, `TxId` has been implemented without relying on `Serialisable` (by using cardano-api).